src/Configuration.php

```diff
@@ -206,6 +206,11 @@ public function skipPermisssionCheck()
         return $this->configuration['security']['skipPermissionCheck'] ?? false;
     }
 
+    public function disableVersionedRequests(): bool|string
+    {
+        return $this->configuration['security']['disableVersionedRequests'] ?? false;
```

Comment:
for safety reasons, shouldn't this default to true?

Reply:
Yes, that is probably right!